pom.xml

```diff
@@ -148,6 +142,84 @@
                 <artifactId>groovy-all</artifactId>
                 <version>${groovy-all.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.springframework.cloud</groupId>
+                <artifactId>spring-cloud-dependencies</artifactId>
+                <version>Dalston.RELEASE</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+            <!--When we roll our on version of DefaultHttpFirewall, we can remove these managed spring security dependencies-->
+            <dependency>
+                <groupId>org.springframework.security</groupId>
+                <artifactId>spring-security-web</artifactId>
+                <version>4.2.0.RELEASE</version>

```

Comment:
This dependency along with all other Spring-Security-related ones here have been removed.